### PR TITLE
Game Examples: Updated moved URLs

### PR DESCRIPTION
**Pull request by OnkarRuikar**
## Summary

Some URLs have been moved to new domains/locations. And current URLs are being redirected to the new locations.

### Motivation
Before they decommission the old domains and the URLs break. We need to update the URLs with new locations.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/games/examples/index.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ This page lists a number of impressive web technology demos for you to get inspi
 
 ## Free/demo games
 
-- [Beloola](http://www.beloola.com)
+- [Beloola](https://www.youtube.com/watch?v=pVXr3luEJLc)
   - : WebVR platform to connect passionate people. Experience both available on 2D screens and on VR HMDs (_Settings / Switch to VR mode_).
```
Comment (ddbeck) [inline files/en-us/games/examples/index.md:17]: This seems to no longer be available to use. I think we should drop this one from the list

File: files/en-us/games/examples/index.md [hunk 2]
```diff
@@ -28,9 +28,9 @@ This page lists a number of impressive web technology demos for you to get inspi
 - [HexGL](https://hexgl.bkcore.com/)
   - : A fast-paced, futuristic racing game.
 - [Dead Trigger 2](https://beta.unity3d.com/jonas/DT2/)
-  - : Classic zombie splatter action, made with [Unity3D](https://unity3d.com/).
+  - : Classic zombie splatter action, made with [Unity3D](https://unity.com/).
```
Comment (ddbeck) [inline files/en-us/games/examples/index.md:31]: This seems to be behind a Facebook login now. I'd drop this one too

File: files/en-us/games/examples/index.md [hunk 3]
```diff
@@ -28,9 +28,9 @@ This page lists a number of impressive web technology demos for you to get inspi
...
 - [Angry Bots](https://beta.unity3d.com/jonas/AngryBots/)
-  - : A futuristic Aliens-esque isometric 3rd person shooter demo, made with [Unity3D](https://unity3d.com/).
+  - : A futuristic Aliens-esque isometric 3rd person shooter demo, made with [Unity3D](https://unity.com/).
```
Comment (ddbeck) [inline files/en-us/games/examples/index.md:33]:
```suggestion
  - : A futuristic Aliens-esque isometric 3rd person shooter demo, made with [Unity](https://unity.com/).
```

File: files/en-us/games/examples/index.md [hunk 4]
```diff
@@ -68,7 +68,7 @@ This page lists a number of impressive web technology demos for you to get inspi
 
 ## Commercial games
 
-- [Oort Online](http://oortonline.com/)
+- [Oort Online](https://playboundless.com/)
   - : A MMO exploration, building, and battle game (currently in development.)
```
Comment (ddbeck) [inline files/en-us/games/examples/index.md:72]: This looks to be a paid-for product now, not a free demo. Let's drop it

File: files/en-us/games/examples/index.md [hunk 5]
```diff
@@ -82,7 +82,7 @@ This page lists a number of impressive web technology demos for you to get inspi
   - : 8-bit graphic retro adventure game.
 - [RPG MO](https://data.mo.ee/index2.html)
   - : Isometric MMORPG with similarities to RuneScape Classic and Ultima.
-- [Katan.io](https://katan.io/)
+- [Katan.io](https://colonist.io/)
```
Comment (ddbeck) [inline files/en-us/games/examples/index.md:85]:
```suggestion
- [Colonist](https://colonist.io/)
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
A few suggestions for you. Thank you!

**Comment by OnkarRuikar** [comment]
@ddbeck I've directly modified the file and addressed all review comments in one go. 
So marking all the suggestions resolved.

**Review by ddbeck — COMMENTED** [review]
Looks good. Thank you! 🎉

**Review by ddbeck — APPROVED** [review]

**Pull request merged**